tools/cli/src/mentat_cli/command_parser.rs

```diff
@@ -129,6 +135,15 @@ pub fn command(s: &str) -> Result<Command, cli::Error> {
                         }
                         Ok(Command::Close)
                     });
+                    
+    let schema_parser = string(SCHEMA_COMMAND)
```

Comment:
I tried to generalise `no_arg_parser` to take a command as an arg and then use that to determine how the command was parsed, but I couldn't get the lifetimes right. I think that was due to `CLOSE_COMMAND`, `SCHEMA_COMMAND` and `EXIT_COMMAND` all being static strings, which I then also couldn't use in a `match`. I think there is probably some way to get around it that I just haven't figured out yet. Need to spend some more time with the rust book I think. Will file a follow up bug for it.